lib/graphql-exception.filter.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  ExceptionFilter,
+} from '@nestjs/common';
+import {
+  ApolloError,
+  AuthenticationError,
+  ForbiddenError,
+} from 'apollo-server-express';
```

Comment:
We can't import from the 'apollo-server-express' as this will break Fastify based applications. Can you import from base/core packages (not sure which one exposes these error classes)